File: codeforces/20.nearly_lucky_number/nln.cpp
#include <iostream>
#include <cstring>
/* Author: José Rodolfo */
using namespace std;
int main() {
  string n;
  unsigned int quantity_numbers = 0;
  cin >> n;
  unsigned short int size_n = n.size();
  for (unsigned short position = 0; position < size_n; position++) {
    if (n[position] == '4' || n[position] == '7') {
      quantity_numbers++;
    }
  }
  if (quantity_numbers == 4 || quantity_numbers == 7) {
    cout << "YES" << endl;
  }
  else {
    cout << "NO" << endl;
  }
  return 0;
}

File: codeforces/low_difficulty/800_points/20.nearly_lucky_number_[110A]/nln.py
# Author: José Rodolfo (jric2002)
n = str(input())
quantity_numbers = 0
for position in range(0, len(n)):
  if (n[position] == '4' or n[position] == '7'):
    quantity_numbers += 1
if (quantity_numbers == 4 or quantity_numbers == 7):
  print("YES")
else:
  print("NO")